sensing/autoware_pointcloud_preprocessor/schema/radius_search_2d_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radius Search 2D Outlier Filter Node",
+  "type": "object",
+  "definitions": {
+    "crop_box_filter": {
```

Comment:
Schema validation fails. Should be `radius_search_2d_outlier_filter`.

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8474/commits/6c58687da089b7ae11fadfbab14295a669020a2e